steps/xcode-test-mac/1.2.0/step.yml

```diff
@@ -0,0 +1,93 @@
+title: Xcode Test for Mac
+summary: Run Xcode Test on your Mac OS X app project.
+description: Runs Xcode's `test` action, configured for Mac OS X app projects.
+website: https://github.com/bitrise-steplib/steps-xcode-test-mac
+source_code_url: https://github.com/bitrise-steplib/steps-xcode-test-mac
+support_url: https://github.com/bitrise-steplib/steps-xcode-test-mac/issues
+published_at: !!timestamp 2018-04-09T12:12:26.958936213+02:00
+source:
+  git: https://github.com/bitrise-steplib/steps-xcode-test-mac.git
+  commit: 7c0a138b4e588906e3771671cc45a2e706195c63
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- macos
+type_tags:
+- test
+dependencies:
+- manager: _
+  name: xcode
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-xcode-test-mac
+deps:
+  brew:
+  - name: go
+  check_only:
+  - name: xcode
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- opts:
+    description: |
+      A `.xcodeproj` or `.xcworkspace` path, relative to
+      the Working directory (if specified).
+    is_required: true
+    title: Project (or Workspace) path
+  project_path: $BITRISE_PROJECT_PATH
+- opts:
+    description: |
+      The Scheme to use.
+      **IMPORTANT**: The Scheme have to be marked as __shared__ in Xcode!
+    is_required: true
+    title: Scheme name
+  scheme: $BITRISE_SCHEME
+- destination: null
+  opts:
+    description: |
+      The Destination to use.
+
+      Read more in [Xcodebuild Destination Cheatsheet](http://www.mokacoding.com/blog/xcodebuild-destination-options/).
+
+      Example value: `platform=OS X,arch=x86_64`
+    is_required: false
+    title: Desination name
+- is_clean_build: "yes"
+  opts:
+    is_required: true
+    title: Do a clean Xcode build before testing?
```

Comment:
**XCode**

BTW, I think it would be better this way:
Should a clean XCode build run before testing?
or
Run a clean XCode build before testing?